app/components/common/Footer.vue

```diff
@@ -143,10 +141,23 @@ export default Vue.extend({
             { url: 'https://github.com/codecombat/codecombat', extra: 'GitHub' },
             { url: this.cocoPath('/community'), title: 'nav.community' },
             { url: this.cocoPath('/contribute'), title: 'nav.contribute' },
-            { url: this.cocoPath('/league'), title: 'nav.esports' },
             { url: this.forumLink, title: 'nav.forum', attrs: { target: '_blank' }, hide: me.isStudent() || !me.showForumLink() },
             { url: this.apiLink, title: 'nav.api', attrs: { target: '_blank' }, hide: me.isStudent() }
           ]
+        },{
+          title: 'nav.products',
+          condition: true,
+          lists: [
+            { url: this.ozPath('/'), title: 'nav.ozaria_classroom' },
+            { url: this.cocoPath('/students'), title: 'nav.codecombat_classroom' },
```

Comment:
I don't think we can link directly to /students yet. Instead, since we don't actually have a CodeCombat Classroom page, we should just link to https://codecombat.com/impact.